packages/nimble-components/src/table/index.ts

```diff
@@ -684,37 +683,6 @@ export class Table<
         return tanStackUpdates;
     }
 
-    /** @internal */
-    public handleFocusedCellRecycling(): void {
-        const hadActiveRowOrCellFocus = this.keyboardNavigationManager.hasActiveRowOrCellFocus;
-
-        let tableFocusedElement = this.shadowRoot!.activeElement;
-        while (
-            tableFocusedElement !== null
-            && !(tableFocusedElement instanceof TableCellView)
-        ) {
-            if (tableFocusedElement.shadowRoot) {
-                tableFocusedElement = tableFocusedElement.shadowRoot.activeElement;
-            } else {
-                break;
-            }
-        }
-        if (tableFocusedElement instanceof TableCellView) {
-            tableFocusedElement.focusedRecycleCallback();
-        }
-        if (this.openActionMenuRecordId !== undefined) {
-            const activeRow = this.rowElements.find(
-                row => row instanceof TableRow
-                    && row.recordId === this.openActionMenuRecordId
-            ) as TableRow | undefined;
-            activeRow?.closeOpenActionMenus();
```

Comment:
This code can go away for the same reason `focusedRecycleCallback` does: focusing the cell after a scroll accomplishes the same thing.